modules/kafka-cluster/main.tf

```diff
@@ -1,7 +1,8 @@
-locals {
-  build_cluster_type_basic    = var.cluster_for_production ? [] : ["yes"]
-  build_cluster_type_standard = var.cluster_for_production ? ["yes"] : []
-}
+# locals {
+#   build_cluster_type_basic      = var.cluster_for_production ? [] : ["yes"]
+#   build_cluster_type_standard   = var.cluster_for_production ? ["yes"] : []
+#   build_dedicated_kafka_cluster = var.cluster_for_production ? ["yes"] : []
+# }
 
```

Comment:
```suggestion
```

We can see what was changed from the diff and have git history if needed, thus here we prefer deleting old code rather than leaving it as comments.